R/viz.R

```diff
@@ -2041,7 +2049,9 @@ ashm_multi_rainbow_plot = function(regions_bed,
   p = muts_anno %>%
         ggplot() +
         geom_point(aes(x = start, y = sample_id, colour = classification), alpha = 0.4, size = 0.6) +
-        theme(axis.text.y = element_blank()) +
+        labs(title = "", subtitle = "", x = "", y = "Sample") +
```

Comment:
I am more of a fan of specifying the empty ones in the `theme()` and setting to `element_blank()`. The reason is that when we set it to nothing, it is still processed as "invisible string" and the space for that `` string is allocated on the plot, creating unnecessary white space. Instead, when set through the theme, the title is removed completely and the plotting area has minimum of white space

Reply:
Yes, I follow and agree that this is probably the correct way of doing this. Thanks for the comment.